packages/account-v2/src/index.scss

```diff
@@ -0,0 +1,12 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+
+@import '@deriv/quill-design/dist/quill-design.css';
+@import '@deriv/quill-design/quill-tailwind/styles.css';
+@import '@deriv/quill-design/quill-tailwind/fonts.css';
+
+svg {
+    display: inline;
+    vertical-align: middle;
+}
```

Comment:
Why do we need this?
can you remove the last line from the changes?

Reply:
This is needed as currently the icon alignment is broken while using Quill. As a workaround Tradershub team came up with this